### DISCUSSION TITLE
Please Help my anyone from AdonisJs/Lucid community

### DISCUSSION THREAD

**Discussion by dmostafiz** [community/community#45436] [Other Feature Feedback, Questions, & Ideas]
Hello @everyone

I wasn't able to find any discussion section or a way to reach you. that's why I've opened the issue here.
I really need help.

Please take a bit of time for me,

###my query is
I need to show user registrations every month for the last year.
I did this query in Laravel eloquent but didn't find a solution for how to do this query in Lucid.
Please note! I'm using PostgreSQL and cockroach db  

I need somehow,,  group users count by ( created_at---> month name )  for last year

#User Table   
____________________
id ----- username ---- email ---- created_at ---- updated_at
____________________

#Raw query would be something like that ----> for postgreSQL

SELECT
  to_char(date_trunc('month', created_at), 'YYYY, Month') AS regisrtration_month,
  COUNT(id) AS total_registration,
FROM users
GROUP BY date_trunc('month', created_at)
ORDER BY date_trunc('month', created_at)
_______________________________
But I really need to know how I can do this using Lucid ORM

I'll appreciate any help from you.
Best regards!

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by dmostafiz** [reply]
Thanks.. i already knocked them. but no answer!
